app/components/Footer/index.js

```diff
@@ -64,6 +65,11 @@ const Footer = (props: Props) => (
             />
             <h2>Kontakt</h2>
             <a href="mailto:abakus@abakus.no">abakus@abakus.no</a>
+            {loggedIn && (
+              <p>
+                <Link to="/contact">Anonymt kontaktskjema</Link>
+              </p>
+            )}
```

Comment:
I'm guessing this is just temporary due to the contact form issues?

Reply:
Assuming you're referring to the `loggedIn` check, yes;) It is now mentioned in the contact form issue, so hopefully we'll remember to remove this check when https://github.com/webkom/lego/issues/2345 is resolved.